synapse/lib/agenda.py

```diff
@@ -598,22 +598,34 @@ async def add(self, cdef):
         Persistently adds an appointment
 
         Args:
-            cdef (dict):  May contain the following keys:
-                creator (str):  iden of the creating user
-                iden (str): Iden of the appointment
-                storm (str): storm query to run
-                reqs (Union[None, Dict[TimeUnit, Union[int, Tuple[int]], List[...]):
+            cdef (dict):  Dictionary containing the Cron definition.
+
+        Notes:
+            For values in reqs that are lists and incvals if a list, all combinations of all values (the product) are
```

Comment:
This reads a little clunky, maybe "If the values for req and incvals are both lists, all combinations of all values (the product) are used." or something

Reply:
gotta leave this open. I just moved the existing language around - I'll take a quick swag at rewriting it.